.circleci/config.yml

```diff
@@ -0,0 +1,62 @@
+# Javascript Node CircleCI 2.0 configuration file
+#
+# Check {{ '/2.0/language-javascript/' | docs_url }} for more details
+#
+defaults: &defaults
+  working_directory: ~/repo
+  docker:
+    - image: circleci/node:10
+
+version: 2.1 # use CircleCI 2.1
+jobs: # a collection of steps
+  build: # runs not using Workflows must have a `build` job as entry point
+    <<: *defaults
+    steps: # a collection of executable commands
+      - checkout # special step to check out source code to working directory
+      - restore_cache: # special step to restore the dependency cache
+          # Read about caching dependencies: https://circleci.com/docs/2.0/caching/
+          key: dependency-cache-{{ checksum "yarn.lock" }}
+      - run:
+          name: install-dependencies
+          command: yarn install
+      - save_cache: # special step to save the dependency cache
+          key: dependency-cache-{{ checksum "yarn.lock" }}
+          paths:
+            - ./node_modules
+      - run:
+          name: test
+          command: yarn test
+      - run:
+          name: build
+          command: yarn build
+      - persist_to_workspace:
+          root: .
+          paths:
+            - dist
+  deploy:
+    <<: *defaults
+    steps:
+      - checkout # special step to check out source code to working directory
+      - attach_workspace:
+          at: .
+      - run:
+          name: install-auto
+          command: yarn add -D auto
+      - run:
+          name: Authenticate with registry
+          command: echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > ~/repo/.npmrc
+      - run:
+          name: Publish package
+          command: yarn release
```

Comment:
The `release` script is not present in the `package.json`
You need this:
https://github.com/ardeois/graphql-codegen-typescript-mock-data/blob/7ba4e9a9765f396a79f058b50f21127a25854d6b/package.json#L57-L59

Reply:
@ardeois you probably missed it `"release": "auto changelog && yarn auto:version && yarn auto:publish && auto release"`